js/app/widgets/formattableLabel.js

```diff
@@ -92,7 +92,9 @@ const FormattableLabel = new Knowledge.Class({
         if (!text)
             text = '';
         if (this.use_markup) {
-            Pango.parse_markup(text, -1, '0', null, text);
+            // Pango.parse_markup returns an array where the third
+            // element is the result of the parsing.
+            text = Pango.parse_markup(text, -1, '0', null, text)[2];
```

Comment:
Should we handle broken markup somehow? first return value of false I think? This will just hard error.